wallet/core/src/wasm/message.rs

```diff
@@ -14,6 +14,7 @@ const TS_MESSAGE_TYPES: &'static str = r#"
 export interface ISignMessage {
     message: string;
     privateKey: PrivateKey | string;
+    noAuxRand: boolean;
```

Comment:
Please make this optional here. This ensure any existing usage when this is added